Makefile

```diff
@@ -137,6 +137,12 @@ reproducible-prestate:   ## Builds reproducible-prestate binary
 	make -C ./op-program reproducible-prestate
 .PHONY: reproducible-prestate
 
+# Include any files required for the devnet to build and run. This appears to be the only one that's actually needed.
+DEVNET_CANNON_PRESTATE_FILES := op-program/bin/prestate-proof.json
```

Comment:
The prestate.json and meta.json and all the friends don't seem to be needed. If they are, they would go here.

Reply:
We will need `prestate.json` as part of actually running the devnet (it's just not needed for genesis generation) so probably is worth adding here.